Gemfile

```diff
@@ -218,6 +218,7 @@ end
 unless ENV["APPLIANCE"]
   group :development do
     gem "foreman"
+    gem "graphiql-rails",   "~>1.4"
```

Comment:
Maybe belongs in graphql_api group but it really is a development only dependency, and this is only temporary 🆗